brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/spec/StandardInterfaceLifecycleModifier.java

```diff
@@ -34,17 +43,17 @@ public void apply(EntitySpec<?> entitySpec, String nodeId, ToscaApplication tosc
         }
         // If the entity spec is of type VanillaSoftwareProcess, we assume that it's running. The operations should
         // then take care of setting up the correct scripts.
-        entitySpec.configure(VanillaSoftwareProcess.LAUNCH_COMMAND, "true");
-        entitySpec.configure(VanillaSoftwareProcess.STOP_COMMAND, "true");
-        entitySpec.configure(VanillaSoftwareProcess.CHECK_RUNNING_COMMAND, "true");
+        entitySpec.configure(VanillaSoftwareProcess.LAUNCH_COMMAND.getName(), "true");
```

Comment:
Interesting question: if you look at EntitySpec#configure(String, Object) you'll see it puts it into the _flags_ Map, but if you use EntitySpec#configure(ConfigKey<V>, V) it puts it into the _config_ Map!  And if we don't use the first we can't put int the DeferredSupplier later on, and if we use both then it all goes wrong.